### DISCUSSION TITLE
Add achievement list to GitHub application

### DISCUSSION THREAD

**Discussion by m3r1v3** [community/community#19279] [Mobile] [answered]
Recently, on GitHub was added new achievement, but you can see them only on website. I think achievement list in user profile in GitHub mobile application will be useful

**Comment by melvincwng** [comment, marked as answer]
@merive, honestly that's a very good suggestion!

Would be cool to see achievements in GitHub Mobile app too :)

**Reply by m3r1v3** [reply]
Has been released in v1. 94 🎉

**Reply by octopussy-maker** [reply]
Stay Safe

**Comment by octopussy-maker** [comment]
Chillin like a ghost .Octopussy maker here . Lets grab coffee at Starbucks .

**Reply by m3r1v3** [reply]
Wtf it's means?
